ui/com/msg-view/summary.jsx

```diff
@@ -11,7 +11,8 @@ import u from '../../lib/util'
 export default class Summary extends React.Component {
   onClick(e) {
     // make sure the user didnt click on a link
-    for (var i=0; i < e.nativeEvent.path.length; i++) {
+    var pathLen = e.nativeEvent.path ? e.nativeEvent.path.length : 0
```

Comment:
I dont think we can just default to 0; that would cause this code to fail. This piece would have to be rewritten to recurse up the nodes by following `.parentNode`.